cli/src/commands/create/source.rs

```diff
@@ -29,6 +29,10 @@ pub struct CreateSourceArgs {
     #[structopt(long = "bucket")]
     /// Bucket to pull emails from.
     bucket: Option<BucketIdentifier>,
+
+    #[structopt(long = "source-type")]
+    /// Set the source type of the new source
+    source_type: Option<String>,
```

Comment:
This should now be
```suggestion
    source_type: Option<SourceType>,
```
As mentioned above, `structopt` would call `from_str` on `SourceType` to parse a value from the string you provide as an argument.

Currentlym I don't think you're currently using `SourceType`.

Reply:
I am using `SourceType` I promise! I even had a test failing cause I had implemented it wrong at first. But yeah, I forgot here, oops! Will fix.